src/main/java/li/cil/tis3d/common/tileentity/ComputerTileEntity.java

```diff
@@ -168,7 +167,7 @@ public void onDataPacket(final Connection manager, final ClientboundBlockEntityD
     public ClientboundBlockEntityDataPacket getUpdatePacket() {
         final CompoundTag tag = new CompoundTag();
         saveClient(tag);
```

Comment:
Can remove those two lines now, right?

Reply:
apparently yes. i was cautious about deleting things i didn't know for SURE were now irrelevant, since i didn't write most of this code and i didn't understand the history behind it. XD